src/components/biggive-article-card/test/biggive-article-card.spec.tsx

```diff
@@ -19,7 +19,7 @@ describe('biggive-article-card', () => {
                      <a></a>
                    </h3>
                    <div class="image-group">
-                     <div class="image-wrap" style="background-image: url('null');">
```

Comment:
I don't really like this undefined, but it's probably not any worse than the null that was there before. I guess ideally we'd pass an actual prop into the component from the test.